oci.go

```diff
@@ -34,13 +33,13 @@ const (
 	cgroupsMountType = "cgroup"
 )
 
-var (
-	errNeedLinuxResource = errors.New("Linux resource cannot be empty")
-)
-
 var cgroupsMemDirPath = "/sys/fs/cgroup"
 
 func containerExists(containerID string) (bool, error) {
+	if containerID == "" {
```

Comment:
I can't see the value of having a special case for "". containerExists() will already return false. Somethig that would add value though is returning an error when returning false in this function, right now we `return false, nil` and that's clearly a bug.